src/libraries/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -18,10 +18,8 @@
     <NoWarn Condition="'$(TargetFramework)' == 'netstandard2.0' or $(TargetFramework.StartsWith('net4'))">$(NoWarn);nullable</NoWarn>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="$(CommonPath)System\Runtime\CompilerServices\PreserveDependencyAttribute.cs"
-             Link="Common\System\Runtime\CompilerServices\PreserveDependencyAttribute.cs" />
-    <Compile Include="$(CommonPath)System\HexConverter.cs"
-             Link="Common\System\HexConverter.cs" />
+    <Compile Include="$(CommonPath)System\Runtime\CompilerServices\PreserveDependencyAttribute.cs" Link="Common\System\Runtime\CompilerServices\PreserveDependencyAttribute.cs" />
```

Comment:
Nit: Undo formatting changes?

Reply:
I'm not sure if we have some VS formatting rules in the repo, but I've been fighting this automatic change when saving .csproj files over multiple PRs now. The same thing happens in [this PR](https://github.com/dotnet/runtime/pull/36829/files#diff-bc2a8297e6616644e3d9970f7e8d41a9R21-R22). I think I'll let it win this time.